src/apps/answers/domain/answers.py

```diff
@@ -41,6 +41,10 @@ class Answer(InternalModel):
     activity_history_id: str
     respondent_id: uuid.UUID | None
     is_flow_completed: bool | None = False
+    target_subject_id: uuid.UUID | None = None
+    source_subject_id: uuid.UUID | None = None
+    input_subject_id: uuid.UUID | None = None
+    relation: str | None = None
```

Comment:
This doesn't look like it's related to the change in Report Server [M2-7093](https://mindlogger.atlassian.net/browse/M2-7093). @rcmerlo any concerns with this change to the Answer class?

Reply:
This change is because this PR changes query to fetch answers from AnswerCRUD().get_by_submit_id() which returns AnswerSchema to  AnswerCRUD().get_list() which return Answers and code needs target_subject_id to get user_info, see bellow this change in service.py file